export const throwCustomError = (errorMessage) => {
  throw new Error(`[ERROR] ${errorMessage}`);
};
